public/v4/apps/airflow.yml

```diff
@@ -0,0 +1,218 @@
+          description: Do you want to use startTLS? (yes/no)
+        - id: $$cap_smtp_ssl
+          label: Use SSL?
+          defaultValue: 'no'
+          description: Check the tag version on https://hub.docker.com/r/bitnami/airflow-scheduler
+        - id: $$cap_smtp_user
+          label: SMTP User
+          description: Check the tag version on https://hub.docker.com/r/bitnami/airflow-scheduler
+        - id: $$cap_smtp_password
+          label: SMTP Password
+          description: Check the tag version on https://hub.docker.com/r/bitnami/airflow-scheduler
+        - id: $$cap_smtp_mail_from
+          label: Mail from address
+          description: Specofy who the email is from
+
+services:
+    $$cap_appname-db:
+        restart: always
+        image: 'bitnami/postgresql:$$cap_postgresql_version'
+        user:
+        environment:
+            POSTGRESQL_DATABASE: $$cap_postgresql_name
+            POSTGRESQL_USERNAME: $$cap_postgresql_user
+            POSTGRESQL_PASSWORD: $$cap_postgresql_password
+        volumes:
+            - $$cap_appname-postgresql-persistance:/bitnami
+        caproverExtra:
+            notExposeAsWebApp: true
+
+    $$cap_appname-redis:
+        restart: always
+        image: 'bitnami/redis:$$cap_redis_version'
+        user:
+        environment:
+            REDIS_PASSWORD: $$cap_redis_password
+        volumes:
+            - $$cap_appname-redis-persistance:/bitnami
+        caproverExtra:
+            notExposeAsWebApp: true
+
+    $$cap_appname:
+        restart: always
+        depends_on:
+            - $$cap_appname-db
+            - $$cap_appname-redis
+        image: 'bitnami/airflow:$$cap_airflow_version'
+        user:
+        environment:
+            AIRFLOW_EXECUTOR: CeleryExecutor
+            AIRFLOW_DATABASE_HOST: srv-captain--$$cap_appname-db
+            AIRFLOW_DATABASE_NAME: $$cap_postgresql_name
+            AIRFLOW_DATABASE_USERNAME: $$cap_postgresql_user
+            AIRFLOW_DATABASE_PASSWORD: $$cap_postgresql_password
+            REDIS_HOST: srv-captain--$$cap_appname-redis
+            REDIS_PASSWORD: $$cap_redis_password
+            AIRFLOW_WEBSERVER_HOST: srv-captain--$$cap_appname
+            AIRFLOW_BASE_URL: https://$$cap_appname.$$cap_root_domain/
+            AIRFLOW_SECRET_KEY: $$cap_airflow_secret_key
+            AIRFLOW_LOAD_EXAMPLES: $$cap_airflow_examples
+            AIRFLOW_USERNAME: $$cap_airflow_user
+            AIRFLOW_PASSWORD: $$cap_airflow_password
+            AIRFLOW_EMAIL: $$cap_airflow_email
+            AIRFLOW__SMTP__SMTP_HOST: $$cap_smtp_host
+            AIRFLOW__SMTP__SMTP_PORT: $$cap_smtp_port
+            AIRFLOW__SMTP__SMTP_STARTTLS: $$cap_smtp_starttls
+            AIRFLOW__SMTP__SMTP_SSL: $$cap_smtp_ssl
+            AIRFLOW__SMTP__SMTP_USER: $$cap_smtp_user
+            AIRFLOW__SMTP__SMTP_PASSWORD: $$cap_smtp_password
+            AIRFLOW__SMTP__SMTP_MAIL_FROM: $$cap_smtp_mail_from
+        ports:
+            - '8080:8080'
```

Comment:
Why does it need a mapped port if it's already added as an HTTP container port?

Reply:
hey @githubsaturn 
That is the way the suggested docker compose file is set, I assumed I needed to define that port for that service, then tell caprover to map it to http and https ports.

My assumption is those two things are independent and both need to be explicitly defined.
Isn't that the case?

Reply:
Yes, when you've added the port as the container port, the mapping is typically not needed anymore. I suggest removing it and do a quick test to make sure it works.